src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -5316,7 +5316,7 @@ private void RemoveFromRemoteRunspaceRunningList()
         private RemoteRunspacePoolInternal GetRemoteRunspacePoolInternal()
         {
             RunspacePool runspacePool = _rsConnection as RunspacePool;
-            return (runspacePool != null) ? (runspacePool.RemoteRunspacePoolInternal) : null;
+            return runspacePool?.RemoteRunspacePoolInternal;
```

Comment:
This does not look like it is related to bumping .NET version. Probably better to remove this from this PR.

Reply:
This is due to this issue i reported: https://github.com/dotnet/roslyn/issues/51849